Northstar.Custom/mod.json

```diff
@@ -1,7 +1,7 @@
 {
 	"Name" : "Northstar.Custom",
 	"Description": "(ideally) Polished custom content for Northstar",
-	"Version": "1.0.0",
+	"Version": "2.0.0",
```

Comment:
Version change should only be done on release by org member IMO.

On that note, @abarichello probably wanna set that to `0.0.0` for all mods as well and update the value via CI on release.